Gemfile.lock

```diff
@@ -257,3 +276,6 @@ DEPENDENCIES
   uglifier (>= 1.3.0)
   unicorn (= 4.8.2)
   webmock (~> 2.1.0)
+
+BUNDLED WITH
+   1.12.5
```

Comment:
Ruby 2.3.1 which this project uses is pinned to bundler `1.11.2` (see https://github.com/alphagov/govuk-puppet/blob/9a17813a00e18ad6c22db48ceda24d233d404b24/modules/govuk_rbenv/manifests/all.pp#L56) so this line shouldn't read `1.12.5`.